### PR TITLE
Create admin API to get task queue tasks

### PR DESCRIPTION
**Pull request by feedmeapples**
**What changed?**

Created an admin API that that allows to get task-queue tasks

**Why?**

simplifie get task queue tasks in tctl admin command by removing the need for direct DB connection params

**How did you test it?**

`./tctl admin taskqueue list_tasks --namespace default --taskqueue open-n-closed`

**Potential risks**

adds a new API, no risks for existing functionality 

**Is hotfix candidate?**

### CODE REVIEW

File: proto/internal/temporal/server/api/adminservice/v1/request_response.proto [hunk 1]
```diff
@@ -360,3 +362,16 @@ message ResendReplicationTasksRequest {
 
 message ResendReplicationTasksResponse {
 }
+
+message GetTaskQueueTasksRequest {
```
Comment (wxing1292) [inline proto/internal/temporal/server/api/adminservice/v1/request_response.proto:366]: GetTaskQueueTasksRequest -> ListTaskQueueTasksRequest & add a pagination token?

File: proto/internal/temporal/server/api/adminservice/v1/request_response.proto [hunk 2]
```diff
@@ -360,3 +362,16 @@ message ResendReplicationTasksRequest {
...
+    string namespace_id = 1;
```
Comment (yiminc) [inline proto/internal/temporal/server/api/adminservice/v1/request_response.proto:367]: I think we should use namespace.

Comment (feedmeapples) [reply proto/internal/temporal/server/api/adminservice/v1/request_response.proto]: resolved

File: proto/internal/temporal/server/api/adminservice/v1/request_response.proto (hunk 1 above)
Comment (feedmeapples) [reply proto/internal/temporal/server/api/adminservice/v1/request_response.proto:429]: afaict this also requires changes in persistence right so it returns a page token? i'm opening a ticket for refactoring

**Review by yiminc — APPROVED** [review]

**Pull request merged**